.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
+OPENAI_API_KEY=
```

Comment:
Consider adding validation or error handling when these environment variables are accessed in the code to provide clear feedback if they are not set, which would aid in debugging.